src/App.test.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+const puppeteer = require('puppeteer');
+const faker = require('faker');
+
+const person = {
+  name: faker.name.firstName() + ' ' + faker.name.lastName(),
+  email: "c@lvin.com",
+  phone: faker.phone.phoneNumber(),
+  message: faker.random.words(),
+  password: "password"
+};
+
+
+
+let browser;
+let page;
+const width = 1920;
+const height = 1080;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch({
+    headless: false,
+    slowMo: 80,
```

Comment:
This should probably default to having no `slowMo` or browser opening to run tests faster (you shouldn't need to see exactly what is going on for other people's component tests that you didn't touch, just to see if it didn't break - if it does break, then you can turn this back on to see what happened).

Reply:
if you don't need to browser to open you can just set `headless: true`. slowMo is slowing the speed of puppeteer by millisecond. I've only had it on for the login demonstration.